src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj

```diff
@@ -47,6 +48,8 @@
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.CodeLens" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.CSharp" />
+    <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Implementation" />
+    <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.LiveShare" />
```

Comment:
why is this necessary?

Reply:
These are local projects in this solution. Added so they have access to the VisualStudioMefHostServices type in the shared layer. If we were willing to pick up a vs-mef dependency even lower (features layer) we could avoid this.